File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/cot/Java/C/s466963472_comments_removed.c
#include <stdio.h>
#include <stdbool.h>

int d;

bool read() {
    if(scanf("%d", &d) != 1) return false;
    return true;
}

int solve() {
    int ans = 0;
    for(int i = 1; i * d < 600; i++) {
        int x = i * d;
        int y = x * x;
        ans += y * d;
    }
    return ans;
}

int main() {
    while(read()) {
        printf("%d\n", solve());
    }
    // End of Code
}

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/cot/Java/Go/s466963472_comments_removed.go
package main

import (
	"fmt"
	"bufio"
	"os"
	"strconv"
	"strings"
)

var d int
var scanner *bufio.Scanner

func read() bool {
	if scanner.Scan() {
		line := scanner.Text()
		if len(line) == 0 {
			return false
		}
		var err error
		d, err = strconv.Atoi(strings.TrimSpace(line))
		return err == nil
	}
	return false
}

func solve() int {
	ans := 0
	for i := 1; i*d < 600; i++ {
		x := i * d
		y := x * x
		ans += y * d
	}
	return ans
}

func main() {
	scanner = bufio.NewScanner(os.Stdin)
	for read() {
		fmt.Println(solve())
	}
}
// End of Code
